### ISSUE TITLE
Can someone test this with Google Assistant?

### ISSUE THREAD

**Issue by UbhiTS** [UbhiTS/ad-alexatalkingclock#11]

**Comment by 5and0** [comment]
Happy to @UbhiTS. What do you need? Adding another media player as an Alexa doesn’t seem to work. Am I missing something? Assume you just have to make a standard TTS call.

**Comment by avd706** [comment]
Yeah. Added the google group and nothing, except I find the tts .mp3s in the confit folder.

**Comment by UbhiTS** [comment]
Okay, I'll update the code, is it possible we can do a remote debug on your machine online

**Comment by avd706** [comment]
thanks... let me know.
I'm still troubleshooting to see if its in issue here

**Comment by avd706** [comment]
seems the issue isits using alexa notifications, not tts services.

**Comment by UbhiTS** [comment]
sorry guys, I've been off the platform for a year. but back now and will start work on all issues. thanks for all your patience. apologies for the delay
